test/models/WebsitePaymentsModel/test.js

```diff
@@ -0,0 +1,141 @@
+const {
+  expect, WebsitePaymentsModel, faker, dropDatabase,
+} = require('test/testHelper');
+const { WebsitePaymentsFactory } = require('test/factories');
+const { PAYMENT_TYPES } = require('../../../constants/sitesConstants');
+
+describe('WebsitePaymentsModel', () => {
+  describe('On find', () => {
+    let host, condition, sort, websitePayments;
+    beforeEach(async () => {
+      await dropDatabase();
+      host = faker.random.string();
+      condition = { host };
+      sort = { createAt: 1 };
+
+      const websitePayment1 = {
+        host,
+        type: PAYMENT_TYPES.WRITE_OFF,
+        amount: 10,
+      };
+      const websitePayment2 = {
+        host,
+        type: PAYMENT_TYPES.REFUND,
+        amount: 15,
+      };
+      const websitePayment3 = {
+        host,
+        type: PAYMENT_TYPES.WRITE_OFF,
+        amount: 20,
+      };
+      const websitePayment4 = {
+        type: PAYMENT_TYPES.REFUND,
+        amount: 13,
+      };
+      websitePayments = [websitePayment1, websitePayment2, websitePayment3, websitePayment4];
+      for (const websitePayment of websitePayments) {
+        await WebsitePaymentsFactory.Create({ ...websitePayment });
+      }
+    });
+
+    it('Should return array 3 elements long', async () => {
+      const { result } = await WebsitePaymentsModel.find({ condition, sort });
+      expect(result).to.have.length(3);
+    });
+    it('Should return array without one element and when searching not to find it', async () => {
+      const { result } = await WebsitePaymentsModel.find({ condition, sort });
+      expect(result).to.not.include(websitePayments[3]);
+    });
+    it('Should return array whose first element will contain the smallest amount & the last largest', async () => {
+      const { result } = await WebsitePaymentsModel.find({ condition, sort });
+      expect(result[0].amount).to.be.eq(websitePayments[0].amount);
+      expect(result[2].amount).to.be.eq(websitePayments[2].amount);
+    });
+    it('Should check that the result will be undefined', async () => {
+      const { error } = await WebsitePaymentsModel.find(faker.random.string());
+      expect(error).to.be.undefined;
+    });
```

Comment:
what are you testing here?
add test error to be exist
add sort descending test
add host check for result array